### PR TITLE
fix select token responsive

### PR DESCRIPTION
**Pull request by dousdogukan**
### Description

Removing the token selection restriction and fixing the responsive issues of Select in addProduct and editProduct.

### Checklist

- [x] Tested changes locally.
- [ ] Ensured code follows project standards.
- [ ] Updated documentation.
- [ ] Added necessary tests.

### Additional Notes

for #34

### CODE REVIEW

File: src/components/CSelect/index.tsx [hunk 1]
```diff
@@ -20,27 +20,49 @@ type CSelectProps = {
   options: OptionType[];
   onChange?: (value: MultiSelectType) => void;
   value?: MultiSelectType;
+  initialEditProductSelectedLength?: number;
```
Comment (ceydae) [inline src/components/CSelect/index.tsx:23]: explain this to me

**Review by ceydae — CHANGES_REQUESTED** [review]

File: src/components/CSelect/index.tsx (hunk 1 above)
Comment (ruveyda-kisla) [reply src/components/CSelect/index.tsx:23]: @dousdogukan

File: src/components/CSelect/index.tsx [hunk 2]
```diff
@@ -20,27 +20,47 @@ type CSelectProps = {
   options: OptionType[];
   onChange?: (value: MultiSelectType) => void;
   value?: MultiSelectType;
+  initialValuesLength?: number;
+  isCloseModal?: boolean;
```
Comment (ceydae) [inline src/components/CSelect/index.tsx:24]: rename to isModalClose

File: src/components/CSelect/selectCustomStyles.ts [hunk 3]
```diff
@@ -1,7 +1,10 @@
 import { OptionType } from '@/models';
 import { StylesConfig } from 'react-select';
 
-const customStyles = (): StylesConfig<OptionType, false> => ({
+const customStyles = (
+  ItemsSelectedLength: number,
```
Comment (ceydae) [inline src/components/CSelect/selectCustomStyles.ts:5]: selectedItemsLength

Why is the first letter capital?

**Review by ceydae — CHANGES_REQUESTED** [review]

**Pull request merged**